wizard/views/overview.py

```diff
@@ -0,0 +1,4 @@
+
+
+def render_overview():
```

Comment:
If you rebase, you should be able to format using `black`.

Reply:
Done